### ISSUE TITLE
Simulator throws "Timestep too large" when truncating Norne (one or two report steps)

### ISSUE THREAD

**Issue by flikka** [OPM/opm-autodiff#407]
Setting up a "start norne" job at Jenkins, and notice when I truncate after the first or second report step, I get an exception. Including the third (and so on) works fine.

The might be a sensible reason for this, but I'm posting this in any case. Link to output with two reportsteps: https://ci.opm-project.org/job/Norne-sanity-first-steps-only/15/console

**Comment by bska** [comment]
Just some additional information: The actual message appears to be thrown from the parser, specifically method `Opm::Schedule::getWells(size_t timeStep)` which looks like
```C++
std::vector<WellConstPtr> Schedule::getWells(size_t timeStep) const {
    if (timeStep >= m_timeMap->size()) {
        throw std::invalid_argument("Timestep to large");
    }

    std::vector<WellConstPtr> wells;
    // ...
    return wells;
}
```
Just from the check, I'd say that the message is a little misleading.  I'd probably say something along the lines of
```
Time step number too high
```
or
```
Time step ID too high
```
I have no real idea what's causing this issue.

**Comment by atgeirr** [comment]
This should be fixed in OPM/opm-core#905.

**Comment by flikka** [comment]
This is indeed fixed @atgeirr - good. Closing.
